LeaderboardBackend/Models/Requests/LeaderboardRequests.cs

```diff
@@ -12,19 +15,26 @@ public record CreateLeaderboardRequest
     ///     The display name of the `Leaderboard` to create.
     /// </summary>
     /// <example>Foo Bar</example>
-    public required string Name { get; set; }
+    [Required]
```

Comment:
Why are we using the attribute here? Does it actually validate after all?

Reply:
For the swagger doc

Reply:
The fact that we have to put this here anyway for swagger is one of the reasons I want to just use data annotations instead of FluentValidation.